src/Domain/Entity/Promotion.php

```diff
@@ -7,6 +7,9 @@
 
 class Promotion
 {
+    /** @var  bool */
+    private $isSuperPromotion;
```

Comment:
I find it easier to scan if the properties are in the same order as they were passed into the constructor